Marlin/pins_RAMPS_14.h

```diff
@@ -247,7 +248,8 @@
     #endif
   #else // !NEWPANEL (Old-style panel with shift register)
 
-    #define BEEPER_PIN 33   // No Beeper added
+    // No Beeper added
+    #define BEEPER_PIN 33
```

Comment:
Do you not like end-of-line comments? I'm not sure that these comments are vital. But end-of-line comments give us more lines on-screen at once, and they are visible in the results when you use Find in your IDE or text editor.